js/StandardLibraryIntellisense.ts

```diff
@@ -27,6 +33,15 @@ class Sketch {
     AddWire (wire                : oc. TopoDS_Wire) : Sketch;
     
     LineTo  (nextPoint           : number[]       ) : Sketch;
+    HorizontalLineTo(xNextPoint: number): Sketch;
+    HorizontalLineOfLength(xLength: number): Sketch;
+    VerticalLineTo(yNextPoint: number): Sketch;
+    VerticalLineOfLength(yLength: number): Sketch;
+    AngledLineOf(angle: number, lengthOptions: {ofLength?: number, ofVerticalLength?: number, ofHorizontalLength?: number}): Sketch;
+    AngledLineTo(angle: number, point: {toXPoint?: number, toYPoint?: number}): Sketch;
+    Tag(tagName: string): Sketch;
+    AngledLineToTaggedLineIntersect(angle: number, tagName: string, offset: number): Sketch;
+    TangentialArcToAngle(radius: number, angle: number, options?: {obtuse?: boolean, arcType?: ArcType}): Sketch;
```

Comment:
I feel a bit conflicted about this one, I added specifically for the below usecase.
![image](https://user-images.githubusercontent.com/29681384/104169587-ea9b5800-5453-11eb-9507-5104c5531ecb.png)
Where there's a transition of two lines (not at 90 degs) that's filleted but the dimension is given at the start of the fillet, not the intersection of the two straight lines. This makes it difficult to calculate where the second line is in 2d space. But I'm not sure if this is too edge case to deserve it's own function. I'm not sure how common this would be.

Reply:
`ArcType` is a enum (well union type in typescript) but effectively there are for different strings allowed here: `'shortest' | 'longest' | 'clockwise' | 'counterCW'` depending on the geometry shortest or longest will be the same as clockwise or counterCW, but thought having all four made it easier for the user to describe what kind of arc they wanted.
Using this example again, if the sketch is being constructed in the direction of the arrow, 'shortest' or `counterCW` would give the same result.
![image](https://user-images.githubusercontent.com/29681384/104171441-c68d4600-5456-11eb-9e0c-282766c289a7.png)